pkg/daemon/ceph/osd/agent.go

```diff
@@ -92,22 +92,25 @@ func (a *OsdAgent) configureDirs(context *clusterd.Context, dirs map[string]int)
 	succeeded := 0
 	var lastErr error
 	for dirPath, osdID := range dirs {
-		config := &osdConfig{id: osdID, configRoot: dirPath, dir: true, storeConfig: a.storeConfig,
+		cfg := &osdConfig{id: osdID, configRoot: dirPath, dir: true, storeConfig: a.storeConfig,
 			kv: a.kv, storeName: config.GetConfigStoreName(a.nodeName)}
 
-		if config.id == unassignedOSDID {
+		// Force all new directory-based OSDs to be filestore
```

Comment:
Is this coherent with the commit? I thought the commit was disabling the creation of new fs osds?

Reply:
Yes, it is coherent. This is disabling creation of new disk-based filestore osds, but we must continue to support dir-based OSDs (always filestore) for CI testing.

Reply:
Ok